src/components/presentational/LabResultWithSparkline/LabResultWithSparkline.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import { format, parseISO } from "date-fns";
+import "./LabResultWithSparkline.css"
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+
+export interface LabResultValue {
+    Type: string,
+    MostRecentValue: string,
+    AcuityHighlight: string,
+    MostRecentDate: string,
+    NormalRangeTopY: number,
+    NormalRangeBottomY: number,
+    SparklinePoints: SparklinePoint[],
+    TermInformation?: TermInformation
+}
+
+export interface SparklinePoint {
+    X: number,
+    Y: number
+}
+
+export interface TermInformation {
+    TermFamily: string,
+    TermNamespace: string,
+    TermCode: string
+}
+
+export interface LabResultWithSparklineProps {
+    labResultValue: LabResultValue;
+    onViewTermInfo(termInfo: TermInformation): void;
+}
+
+export default function (props: LabResultWithSparklineProps) {
+    function formatDate(d: any) {
+        return format(parseISO(d), "MM/dd/yy");
+    }
+
+    let sparklineXRange = 70;
+    let sparklineYRange = 40;
```

Comment:
Where do these numbers come from?  They appear to be used as scaling values.  Were they just settled upon from past experience?  Or are they tied in with how the sparkline points are computed?

I'm just wondering if maybe we should be computing them dynamically to fit each lab result's data or if this fixed scaling should work for all (or most all) possible lab result values.

Reply:
it is just the width / height of the SVG element I believe.  I can probably update it to pull it dynamically